Tasks/Yakovenko/Task3/time.py

```diff
@@ -0,0 +1,218 @@
+
+from datetime import datetime
+
+d_mins = {
+00: "ровно",
+ 1: "одна минута",
+ 2: "две минуты",
+ 3: "три минуты",
+ 4: "четыре минуты",
+ 5: "пять минут",
+ 6: "шесть минут",
+ 7: "семь минут",
+ 8: "восемь минут",
+ 9: "девять минут",
+ 10: "десять минут",
+ 11: "одинадцать минут",
+ 12: "двенадцать минут",
+ 13: "тринадцать минут",
+ 14: "четырнадцать минут",
+ 15: "пятнадцать минут",
+ 16: "шестнадцать минут",
+ 17: "семнадцать минут",
+ 18: "восемнадцать минут",
+ 19: "девятнадцать минут",
+ 20: "двадцать минут",
+ 21: "двадцать одна минута",
+ 22: "двадцать две минуты",
+ 23: "двадцать три минуты",
+ 24: "двадцать четыре минуты",
+ 25: "двадцать пять минут",
+ 26: "двадцать шесть минут",
+ 27: "двадцать семь минут",
+ 28: "двадцать восемь минут",
+ 29: "двадцать девять минут",
+ 31: "тридцать одна минута",
+ 32: "тридцать две минуты",
+ 33: "тридцать три минуты",
+ 34: "тридцать четыре минуты",
+ 35: "тридцать пять минут",
+ 36: "тридцать шесть минут",
+ 37: "тридцать семь минут",
+ 38: "тридцать восемь минут",
+ 39: "тридцать девять минут",
+ 40: "сорок минут",
+ 41: "сорок одна минута",
+ 42: "сорок две минуты",
+ 43: "сорок три минуты",
+ 44: "сорок четыре минуты",
+ 45: "без пятнадцати минут",
+ 46: "без четырнадцати минут",
+ 47: "без тринадцати минут",
+ 48: "без двенадцати минут",
+ 49: "без одинадцати минут",
+ 50: "без десяти минут",
+ 51: "без девяти минут",
+ 52: "без восеми минут",
+ 53: "без семи минут",
+ 54: "без шести минут",
+ 55: "без пяти минут",
+ 56: "без четырех минут",
+ 57: "без трех минут",
+ 58: "без двух минут",
+ 59: "без одной минуты",
+ 30: "половина"
+ }
+d_hours = {
+ 00: "первого",
+ 1: "второго",
+ 2: "третьего",
+ 3: "четвертого",
+ 4: "пятого",
+ 5: "шестого",
+ 6: "седьмого",
+ 7: "восьмого",
+ 8: "девятого",
+ 9: "десятого",
+ 10: "одинадцатого",
+ 11: "двенадцатого",
+ 12: "первого",
+ 13: "второго",
+ 14: "третьего",
+ 15: "четвертого",
+ 16: "пятого",
+ 17: "шестого",
+ 18: "седьмого",
+ 19: "восьмого",
+ 20: "девятого",
+ 21: "десятого",
+ 22: "одинадцатого",
+ 23: "двенадцатого",
+}
+d_hours_sharp = {
+ 1: "один час",
+ 2: "два часа",
+ 3: "три часа",
+ 4: "четыре часа",
+ 5: "пять часов",
+ 6: "шесть часов",
+ 7: "семь часов",
+ 8: "восемь часов",
+ 9: "девять часов",
+ 10: "десять часов",
+ 11: "одинадцать часов",
+ 12: "двенадцать часов",
+ 13: "один час",
+ 14: "два часа",
+ 15: "три часа",
+ 16: "четыре часа",
+ 17: "пять часов",
+ 18: "шесть часов",
+ 19: "семь часов",
+ 20: "восемь часов",
+ 21: "девять часов",
+ 22: "десять часов",
+ 23: "одинадцать часов",
+ 00: "полночь"
+}
+d_hours_45 = {
+ 1: "два",
+ 2: "три",
+ 3: "четыре",
+ 4: "пять",
+ 5: "шесть",
+ 6: "семь",
+ 7: "восемь",
+ 8: "девять",
+ 9: "десять",
+ 10: "одинадцать",
+ 11: "двенадцать",
+ 12: "час",
+ 13: "два",
+ 14: "три",
+ 15: "четыре",
+ 16: "пять",
+ 17: "шесть",
+ 18: "семь",
+ 19: "восемь",
+ 20: "девять",
+ 21: "десять",
+ 22: "одинадцать",
+ 23: "двенадцать"
+}
```

Comment:
the data structure is overkill and can be easily simplified and cleared from duplicates, but it's a minor problem